### PR TITLE
fix: DateTimeWidget not timezone sensitive (#813)

### PR DESCRIPTION
**Pull request by bmihelac**
fixes: #942, #813

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -100,3 +100,7 @@
     'root': {
         'handlers': ['console'],
     }}
+
+
+# USE_TZ = True
```
Comment (andrewgy8) [inline tests/settings.py]: Should this be removed?

File: tests/core/tests/test_widgets.py [hunk 2]
```diff
@@ -62,14 +63,11 @@ def test_clean(self):
         self.assertEqual(self.widget.clean("13.08.2012 18:00:00"),
                          self.datetime)
 
-    @override_settings(USE_TZ=True)
+    @override_settings(USE_TZ=True, TIME_ZONE='Europe/Ljubljana')
```
Comment (andrewgy8) [inline tests/core/tests/test_widgets.py]: What do you think about keeping a test for when the timezone is not set in the settings? Unless I'm misunderstanding something, the package should be able to handle both cases?

File: tests/settings.py (hunk 1 above)
Comment (bmihelac) [reply tests/settings.py]: I left it intentionally so someone could test example with timezone. But if it is not clear it should be removed probably.

File: tests/core/tests/test_widgets.py (hunk 2 above)
Comment (bmihelac) [reply tests/core/tests/test_widgets.py]: When timezone is not specified Django use s default one (not remember which one it is). So it is always there, and specifying is more explicit

**Pull request merged**